packages/clay-card/src/CardWithNavigation.tsx

```diff
@@ -64,7 +74,19 @@ export const ClayCardWithNavigation: React.FunctionComponent<IProps> = ({
 			horizontal={horizontal}
 			href={href}
 			interactive
-			onClick={onClick}
+			onClick={onClickCallback}
+			onKeyDown={(event: React.KeyboardEvent) => {
+				if (
+					(event && event.keyCode === KEYCODES.ENTER) ||
+					(event && event.keyCode === KEYCODES.SPACE)
+				) {
+					event.preventDefault();
+
+					if (onClickCallback) {
+						onClickCallback(event);
+					}
+				}
+			}}
```

Comment:
I think we should always call `onKeyDown` callback here.

Reply:
If We change the element to a button, yes.

> The original behaviour is when a button is focused, this element be triggered by SPACE or a ENTER key but when using a div with a role="button" this isn't possible.

Reply:
What I'm saying is, regardless of not going to `button`, we should allow people to add `onKeyDown` listeners here.

Reply:
Oh, gotcha.